Backpack/Shadow/README.md

```diff
@@ -0,0 +1,25 @@
+# Backpack/Shadow
+
+## Installation
+
+In `Podfile` add
+
+```
+pod 'Backpack/Shadow'
+```
+
+and then run `pod install`.
+
+## Usage
+
+`Backpack/Shadow` contains the Backpack shadow definitions in the class `BPKShadow`.
+
+```objective-c
+#import <Backpack/Shadow.h>
+
+BPKShadow *shadowSm = [BPKShadow shadowSm];
+// BPKShadow *shadowLg = [BPKShadow shadowLg];
```

Comment:
why are these commented out?

Reply:
Just to illustrate that they are there, makes it easier to copy the code